nixnet/database/_frame.py

```diff
@@ -239,28 +243,24 @@ def mux_subframes(self):
         return self._mux_subframes
 
     @property
-    def pdu_refs(self):
-        return _props.get_frame_pdu_refs(self._handle)
-
-    @pdu_refs.setter
-    def pdu_refs(self, value):
-        _props.set_frame_pdu_refs(self._handle, value)
-
-    @property
-    def pdu_start_bits(self):
-        return _props.get_frame_pdu_start_bits(self._handle)
-
-    @pdu_start_bits.setter
-    def pdu_start_bits(self, value):
-        _props.set_frame_pdu_start_bits(self._handle, value)
-
-    @property
-    def pdu_update_bits(self):
-        return _props.get_frame_pdu_update_bits(self._handle)
-
-    @pdu_update_bits.setter
-    def pdu_update_bits(self, value):
-        _props.set_frame_pdu_update_bits(self._handle, value)
+    def pdus(self):
+        handles = [_pdu.Pdu(handle)
+                   for handle in _props.get_frame_pdu_refs(self._handle)]
```

Comment:
While acceptable by flake8, I personally discourage this style because it leads to unnecessarily deep nesting, and simple refactors (like renaming `handles`) cause  other lines to change (updating indentation)

Alternatively you can do

```python
handles = [
    _pdu.Pdu(handle)
    for handle in _props.get_frame_pdu_refs(self._handle)]
```

Reply:
Sure. I was sticking with what autopep8 did, I don't have any horse in this fight.

Reply:
I need to get an auto formatter setup.  I hear good things about yapf.  A new one recently came out but I can't find it.